server/text_generation_server/cli.py

```diff
@@ -150,6 +150,34 @@ def download_weights(
         # Convert pytorch weights to safetensors
         utils.convert_files(local_pt_files, local_st_files)
 
+@app.command()
+def quantize(
+    model_id: str,
+    output_dir: str,
+    revision: Optional[str] = None,
+    logger_level: str = "INFO",
+    json_output: bool = False,
+    trust_remote_code: bool = False,
+):
+    extension: str = ".safetensors",
+    # Remove default handler
+    logger.remove()
```

Comment:
I'm not sure how this works. Since you call download_weights below I don't think it's needed.

Reply:
Removed.